db/migrate/20200528214834_create_users.rb

```diff
@@ -0,0 +1,11 @@
+class CreateUsers < ActiveRecord::Migration[5.1]
+  def change
+    create_table :users do |t|
+      t.string :name
```

Comment:
So are we currently putting the first and last name in the name column?

Reply:
Right now we just have the first name - there was a key for the last name, but I noticed my specific last name was blank. I didn't think there would be a use for the last name in our app, but we could definitely add a column if you think its best

Reply:
We just have it getting the first name or 'given_name' attached to the Google account. Do you think we should get the last name or 'family_name' from Google as well?

Reply:
Ah. Makes sense. Our app shouldn't need the last name really